<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HillClimbingWidgetClass</class>
 <widget class="QWidget" name="HillClimbingWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HillClimbingWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QGroupBox" name="hillClimbing_group">
     <property name="title">
      <string>Hill Climbing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="calc_button">
        <property name="text">
         <string>Compute</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="startPos_label">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="startPos_spinBox"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Clear Points</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="dataSet_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Data Set</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QComboBox" name="defaultSet_comboBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Random Set</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Set 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Set 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Set 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Set 4</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="load_button">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load a data set with tab separated values&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Load </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QChartView" name="chartView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>load_button</sender>
   <signal>clicked()</signal>
   <receiver>HillClimbingWidgetClass</receiver>
   <slot>onLoadValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calc_button</sender>
   <signal>clicked()</signal>
   <receiver>HillClimbingWidgetClass</receiver>
   <slot>onRunHillClimbing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultSet_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>HillClimbingWidgetClass</receiver>
   <slot>onLoadDefaultValues(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>HillClimbingWidgetClass</receiver>
   <slot>onResetHillClimbing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLoadValues()</slot>
  <slot>onRunHillClimbing()</slot>
  <slot>onLoadDefaultValues(int)</slot>
  <slot>onResetHillClimbing()</slot>
 </slots>
</ui>
